common/src/main/java/com/sequenceiq/cloudbreak/ccm/endpoint/ServiceEndpointFinder.java

```diff
@@ -0,0 +1,21 @@
+package com.sequenceiq.cloudbreak.ccm.endpoint;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Lookup service for service endpoints.
+ */
+public interface ServiceEndpointFinder {
+
+    /**
+     * Returns the service endpoint for the specified request.
+     *
+     * @param serviceEndpointRequest the service endpoint request
+     * @return the service endpoint for the specified request
+     * @throws ServiceEndpointLookupException if an exception occurs
```

Comment:
Again, this javadoc can be more specific

Reply:
Ditto